pkg/reconciler/revision/resources/meta.go

```diff
@@ -18,17 +18,31 @@ package resources
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"knative.dev/pkg/kmeta"
 	"knative.dev/serving/pkg/apis/serving"
 	v1 "knative.dev/serving/pkg/apis/serving/v1"
 )
 
+var (
+	excludeLabels = sets.NewString(
+		serving.RouteLabelKey,
+		serving.RevisionLabelKey,
+	)
+
+	excludeAnnotations = sets.NewString(
+		serving.RevisionLastPinnedAnnotationKey,
+		serving.RevisionPreservedAnnotationKey,
+		serving.RoutingStateModifiedAnnotationKey,
+		serving.UpdaterAnnotation,
```

Comment:
Can you comment on dropping the `UpdaterAnnotation`?

ie. if we don't care about this annotation on the deployment should we drop the 'CreatorAnnotation` as well?

Reply:
We should keep both.

Reply:
Removed that from this PR. We might consider it separately.

Do we want Creator/Updater on the underlying resources? If the k8s items are updated, this won't be kept up to date.

Reply:
Webhook sets those on service and from there they propagate downwards.

Reply:
We need to propagate this to the PA, but my question is about the corev1.PodTemplateSpec and appsv1.Deployment where we may not want to propagate all of the knative managed ones.

Anyway let's consider that a different time. It doesn't belong in here.